zhmcclient/_session.py

```diff
@@ -1773,7 +1773,6 @@ def wait_for_completion(self, operation_timeout=None):
           :exc:`~zhmcclient.ParseError`
           :exc:`~zhmcclient.ClientAuthError`
           :exc:`~zhmcclient.ServerAuthError`
-          :exc:`~zhmcclient.ConnectionError`
```

Comment:
Please add a change log entry for this change. The change log entry should make it clear that Session.wait_for_completion() now retries in case of connection errors and no longer raises ConnectionError.

Reply:
Done